src/components/Button/Button.tsx

```diff
@@ -137,8 +175,8 @@ Button.propTypes = {
   /** `hollow` buttons have an outline style, and are suitable for dark backgrounds */
   hollow: PropTypes.bool,
   level: PropTypes.oneOf(levels),
+  onClick: PropTypes.func,
```

Comment:
This would apply to other components too, like Addon.
This conversion PR seems like a good place to remove the old prop types if possible, but I understand if you want to defer it for now.